library/library.ml

```diff
@@ -644,10 +644,9 @@ let load_library_todo f =
   let tasks, _, _ = System.marshal_in_segment f ch in
   let (s5 : seg_proofs), _, _ = System.marshal_in_segment f ch in
   close_in ch;
-  if tasks = None then user_err ~hdr:"restart" (str"not a .vio file");
-  if s2 = None then user_err ~hdr:"restart" (str"not a .vio file");
-  if s3 = None then user_err ~hdr:"restart" (str"not a .vio file");
-  if pi3 (Option.get s2) then user_err ~hdr:"restart" (str"not a .vio file");
+  let err = str "not a compilable file" in
```

Comment:
I think this message should be more explicit, who knows what a compilable file is

Reply:
I suggest "the information to check proofs is not present in the file, you need to recompile the original file again."